src/instructlab/model/evaluate.py

```diff
@@ -658,33 +675,51 @@ def evaluate(
                 for i, evaluator in enumerate(evaluators):
                     branch = branches[i]
                     print(f"Evaluating answers for branch {branch}...")
-                    qa_pairs, error_rate = evaluator.judge_answers(api_base)
-                    qa_pairs_and_errors.append((qa_pairs, error_rate))
+                    overall_score, qa_pairs, error_rate = evaluator.judge_answers(
```

Comment:
I assume this needs a corresponding change to the library?

Reply:
Yes, here's the PR for that: https://github.com/instructlab/eval/pull/138

Reply:
This creates an issue of merge order.  To avoid a conflict, you're going to need to change the cli first to allow the extra param in the tuple return.  Then merge the eval pr, then submit merge this pr

Reply:
The other option would be to average the qa_pairs.